jetty9/src/main/resources/loading_pages/default.loading.page.html

```diff
@@ -0,0 +1,188 @@
+		c0,12.4-2.2,24-6.7,34.8c-4.4,10.8-10.5,20.2-18.1,28.3c-7.7,8.1-16.8,14.5-27.3,19.1c-10.5,4.6-21.9,7-34.1,7
+		c-12.2,0-23.5-2.3-34-7s-19.5-11-27.3-19.1c-7.8-8.1-13.9-17.5-18.3-28.3c-4.4-10.8-6.7-22.4-6.7-34.8c0-12.4,2.2-24.1,6.7-35.1
+		c4.4-11,10.5-20.6,18.3-28.8s16.9-14.7,27.3-19.5c10.4-4.8,21.8-7.2,34-7.2C606.7,643.3,618,645.7,628.6,650.4z M616.9,792.1
+		c7-3.3,13.2-7.8,18.5-13.5s9.5-12.4,12.5-20.1c3-7.8,4.5-16,4.5-24.6c0-8.9-1.5-17.3-4.5-25.1c-3-7.9-7.2-14.7-12.5-20.5
+		s-11.5-10.4-18.5-13.8c-7-3.4-14.5-5.2-22.5-5.2s-15.5,1.7-22.5,5.2c-7,3.4-13.1,8-18.3,13.8s-9.3,12.6-12.3,20.5
+		c-3,7.9-4.5,16.3-4.5,25.1c0,8.7,1.5,16.9,4.5,24.6c3,7.8,7.1,14.5,12.3,20.1c5.2,5.7,11.3,10.2,18.3,13.5s14.5,5,22.5,5
+		C602.5,797.1,609.9,795.4,616.9,792.1z"/>
+</g>
+<path class="st0" d="M61,861.9c-4.6,0-9.2-1.1-13.4-3.4c-9.2-4.9-15-14.5-15-25V720.2c0-15.7,12.7-28.3,28.3-28.3
+	s28.3,12.7,28.3,28.3v60.4l90.6-60.4L45.3,630.4c-13-8.7-16.5-26.3-7.9-39.3c8.7-13,26.3-16.5,39.3-7.9l170.1,113.4
+	c7.9,5.3,12.6,14.1,12.6,23.6s-4.7,18.3-12.6,23.6l-170,113.4C72,860.3,66.5,861.9,61,861.9z"/>
+<g>
+	<path class="st0" d="M685.7,612.6c-0.8,1.8-1.8,3.3-3.1,4.6c-1.3,1.3-2.8,2.3-4.6,3.1c-1.8,0.8-3.6,1.1-5.6,1.1s-3.9-0.4-5.6-1.1
+		s-3.3-1.8-4.6-3.1c-1.3-1.3-2.3-2.8-3.1-4.6S658,609,658,607s0.4-3.9,1.1-5.6c0.8-1.7,1.8-3.3,3.1-4.6c1.3-1.3,2.8-2.3,4.6-3.1
+		c1.8-0.8,3.6-1.1,5.6-1.1s3.9,0.4,5.6,1.1s3.3,1.8,4.6,3.1c1.3,1.3,2.3,2.8,3.1,4.6c0.8,1.7,1.1,3.6,1.1,5.6
+		C686.8,609,686.4,610.9,685.7,612.6z M683,602.4c-0.6-1.4-1.4-2.7-2.5-3.7c-1-1.1-2.3-1.9-3.7-2.5c-1.4-0.6-2.9-0.9-4.5-0.9
+		s-3.1,0.3-4.5,0.9c-1.4,0.6-2.6,1.5-3.7,2.5c-1,1.1-1.9,2.3-2.5,3.7s-0.9,3-0.9,4.6c0,1.6,0.3,3.2,0.9,4.6c0.6,1.4,1.4,2.7,2.5,3.8
+		c1,1.1,2.3,1.9,3.7,2.5c1.4,0.6,2.9,0.9,4.5,0.9s3.1-0.3,4.5-0.9c1.4-0.6,2.6-1.5,3.7-2.5c1-1.1,1.9-2.3,2.5-3.8
+		c0.6-1.4,0.9-3,0.9-4.6C683.9,605.4,683.6,603.8,683,602.4z M676.2,614.6l-2.9-5.6h-3.9v5.5h-3v-15.9h7.6c1.7,0,3,0.5,4,1.4
+		c1,0.9,1.5,2.2,1.5,3.8c0,1.1-0.3,2.1-0.9,3c-0.6,0.9-1.4,1.5-2.4,1.8l3.4,5.9h-3.4V614.6z M674.1,606.3c0.6,0,1.2-0.2,1.7-0.5
+		c0.5-0.4,0.8-1,0.8-1.9c0-0.9-0.2-1.5-0.7-1.9c-0.4-0.4-1-0.5-1.8-0.5h-4.7v4.9L674.1,606.3L674.1,606.3z"/>
+</g>
+</svg>
+</div>
+
+<div class="section-by-status" id="status-STARTING">
+  GoCD server is starting. Please wait ...
+</div>
+
+<div class="section-by-status" id="status-STARTED" style="display: none;">
+  GoCD server has started. You will be automatically redirected to it in a few seconds. <a href="#" onclick="window.location.reload(true);">Go directly to the GoCD server.</a>
+</div>
+
+<div class="section-by-status" id="status-UNKNOWN" style="display: none;">
+  Unable to contact the GoCD server. It might not have started. Please check the server logs.
+</div>
+
+<div id="update-check-message"></div>
+</body>
+
+<script type="text/javascript">
+  // https://github.com/yanatan16/nanoajax/blob/cc89609d8f4a17a346b27e2f85c18e14de18a76b/nanoajax.min.js
```

Comment:
This is MIT licensed and is simple enough to read the whole source of, un-minified. So, I picked it. If not, I could have written this using XMLHttpRequest directly too. Let me know if you want to change this.